mailutil.py

```diff
@@ -127,6 +127,11 @@ def _send_mail(
     )
 
     recipients.update(cc_recipients)
+
+    if email_cfg.has_blacklist():
```

Comment:
rather add an attr `blacklist_entries`. booleans are almost never a good idea

OR (pbly even better): add a method `filter_recipients` to the cfg model class